src/programs/example.tsx

```diff
@@ -0,0 +1,73 @@
+const jsonVal = require('./data.json');
+const _ = require('lodash');
+
+
+interface ConditionProps {
+    [index: number]: any | any[];
```

Comment:
Adding types would be more easier for readability,we should try avoiding the usage of `any`